presidio_evaluator/models/presidio_analyzer_wrapper.py

```diff
@@ -65,6 +66,30 @@ def predict(self, sample: InputSample) -> List[str]:
         )
         return response_tags
 
+    def predict_span(self, sample: InputSample) -> List[Span]:
+        """
+         Returns the predicted spans from the evaluated model
+        :param sample: Sample to be evaluated
+        :return: List of span in Span object
+        """
+        results = self.analyzer_engine.analyze(
+            text=sample.full_text,
+            entities=self.entities,
+            language="en",
```

Comment:
There is another PR which made the language an external dependency. Perhaps we should conform and add language as an argument